src/routers/announcement/index.ts

```diff
@@ -0,0 +1,10 @@
+import express, { type Router } from 'express';
+import addAnnouncement from '../../controllers/announcement/addannouncement';
+import validationData from '../../middlewares/validation';
+import announcementValidation from '../../validations/announcement';
+
+const announcementRouter: Router = express.Router();
+
+announcementRouter.post('/:roleId', validationData(announcementValidation), addAnnouncement);

```

Comment:
what's the purpose of the param roleId here?

Reply:
i just used it because there was no user schema to get the trainer data